static/styles/settings.css

```diff
@@ -1096,8 +1096,9 @@ input[type=checkbox].inline-block {
     cursor: pointer;
 }
 
-#deactivation_user_modal.fade.in {
-    top: calc(50% - 120px);
+.modal.fade.in {
```

Comment:
Does this change now affect all modals?

Reply:
Only modals that utilize the ".modal" class and the ".fade" class, which appears to only be this one, however others that do also use this code (if so) will be fixed as well.

Reply:
:+1:. If you rebase and repush I can merge.